shared/src/business/useCases/pdf/sanitizePdfInteractor.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * sanitizePdfInteractor
+ *
+ * @param {object} applicationContext the application context
+ * @param {object} providers the providers object
+ * @param {string} providers.key the key of the document to sanitize
+ * @returns {boolean} true if successfully sanitized
+ */
+exports.sanitizePdfInteractor = async (applicationContext, { key }) => {
```

Comment:
now that i'm looking at it again with fresh(er) eyes, is "sanitize" still an accurate term for what we're doing in this interactor?

Reply:
The original "sanitize" code that once existed was used to attempt to remove all pdf form interactivity as well as disable script tags.  
Here's jsdoc about what this function (`flatten()`) intends to do:
https://github.com/Hopding/pdf-lib/blob/b8a44bd24b74f4f32456e9809dc4d2d9dc9bf176/src/api/form/PDFForm.ts#L517
I think that although this function is currently only flattening the forms, it has the potential to maybe do other things as well in the future.

Reply:
![Screen Shot 2021-11-10 at 10 02 42 PM](https://user-images.githubusercontent.com/2445917/141235153-eecc71b5-7a45-402b-880a-d12136401bdd.png)
Here's a screenshot of a PDF I uploaded to my local environment that's using `flatten()`.  It was a PDF with an interactive form which I *also* applied my "pdf signature" to.  all those things were flattened and cannot be interacted with or altered from within the pdf viewer.

Reply:
seems like it's more accurately doing a flatten then! i'm fine with leaving it as is, just wanted to note.